Refresh.GameServer/Database/GameDatabaseContext.Assets.cs

```diff
@@ -30,6 +30,34 @@ public partial class GameDatabaseContext // AssetConfiguration
         
         return null;
     }
+
+    public IQueryable<AssetDependencyRelation> GetAssetDependencies(GameAsset asset) 
+        => this._realm.All<AssetDependencyRelation>().Where(a => a.Dependent == asset.AssetHash);
```

Comment:
the return type complicates thing for code that uses this... the dependent is already implied from the `asset` so i think you should just `.AsEnumerable().Select(x => x.Dependency)` and return that instead to get the old behavior back